example/jest.config.js

```diff
@@ -11,5 +11,10 @@ module.exports = {
   },
   setupFilesAfterEnv: [
     "<rootDir>/src/setupJest.ts"
+  ],
+  collectCoverageFrom: [
+    "<rootDir>/src/app/**/*.ts",
+    "!<rootDir>/src/app/**/*.module.ts",
+    "!<rootDir>/src/app/**/test/*"
```

Comment:
where does this `test` folder come from ?

Reply:
I thought it was an Angular Style guide convention to have `test` folders which would contain test utilities. Turns out this is not the case.
Removed the exclusion